visualization/app/codeCharta/state/store/dynamicSettings/dynamicSettings.splitter.ts

```diff
@@ -44,10 +43,10 @@ export function splitDynamicSettingsActions(payload: RecursivePartial<DynamicSet
 		actions.push(splitSearchPatternAction(payload.searchPattern))
 	}
 
-	if (payload.focusedNodePath !== undefined) {
-		actions.push(splitFocusedNodePathAction(payload.focusedNodePath))
-	}
-
+	// is this used?
```

Comment:
I didn't check carefully yet, if this was used. Maybe also a second look? @ce-bo if you have time? :)

Reply:
(note at myself: if not used also delete splitFocusedNodePathAction)

Reply:
it was used and is re-added again